src/main/java/dukeclasses/FastReader.java

```diff
@@ -0,0 +1,61 @@
+package dukeclasses;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+/**
+ * This class handles reading in the input faster.
+ * It is slightly faster than Scanner.
+ */
+public class FastReader {
+    BufferedReader br;
+    StringTokenizer st;
+
+    public FastReader() {
+        br = new BufferedReader(
+                new InputStreamReader(System.in));
+    }
+
+    String next() {
```

Comment:
I think you should include if the method is public / private. If its public you will need to include javadocs for all public methods.

Reply:
I agree with Han Ching's point. It would be good to specify the access restriction for your attributes and methods of this class.

Reply:
Alright thank you!!

Reply:
nice feedback for reviewing.